spec/controllers/spree/admin/variants_controller_spec.rb

```diff
@@ -59,6 +59,19 @@ module Admin
             spree_delete :destroy, id: variant.id, product_id: variant.product.permalink, format: 'js'
             expect(response).to render_template('spree/admin/shared/_destroy')
           end
+
+          it 'refreshes the cache' do
```

Comment:
If you expect these tests to fail, I would set them to `pending` in this commit and change them to `it` in the fixing commit. It makes it even clearer what exactly the commit fixes. And it makes the specs always pass which helps when running git bisect. Do you agree?

Reply:
I've never thought of this approach. Interesting.